### PR TITLE
Feature/hydration

### LINKED ISSUE #22: Refactor Hydration Test
- [x] Utilize beforeEach
- [x] minimize imported data in tests
- [x] check that all tests are working
- [x] check that all tests are actually testing the correct thing
- [x] convert to arrow functions
- [x] check for lets and vars, replace with consts

### PR DESCRIPTION
**Pull request by aemiers**
### Description
* Is this a feature or a fix?
Fix

* Where should the reviewer start?
Hydration test

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Refactored test so that arrow functions are utilized. All tests should pass. I also refactored the data sets at the beginning to that beforeEach is used and there is less data in the test file.

* What outside features or research did you use to implement this fix?
N/A

* How should this be tested?
Make sure all of the tests pass. The last two tests pass, but we need to re-work the functions themselves

### Relevant Screen Shots
***
#### Please review considerations below:
- [x] Follows Turing Style Guidelines
- [x] Changes generate no new warnings or errors
- [x] Checked code for and corrected any spelling errors
- [ ] README has been updated(if applicable)

closes #22

### CODE REVIEW

File: src/Hydration.js [hunk 1]
```diff
@@ -2,23 +2,26 @@ class Hydration {
   constructor(hydrationData) {
     this.hydrationData = hydrationData;
   }
+
   calculateAverageOunces(id) {
     let perDayUserHydration = this.hydrationData.filter((data) => id === data.userID);
```
Comment (alexthompson207) [inline src/Hydration.js:7]: Can this be a `const`

File: src/Hydration.js [hunk 2]
```diff
@@ -2,23 +2,26 @@ class Hydration {
...
-    return perDayUserHydration.reduce((sumSoFar, data) => {
+    return Math.round(perDayUserHydration.reduce((sumSoFar, data) => {
       return sumSoFar += data.numOunces;
-    }, 0) / perDayUserHydration.length;
+    }, 0) / perDayUserHydration.length);
   }
+
   calculateDailyOunces(id, date) {
     let findOuncesByDate = this.hydrationData.find((data) => id === data.userID && date === data.date);
```
Comment (alexthompson207) [inline src/Hydration.js:14]: const?

**Review by alexthompson207 — COMMENTED** [review]
Nice job with refactoring! Left a few comments for you to review. Also you rock

File: src/Hydration.js [hunk 3]
```diff
@@ -2,23 +2,26 @@ class Hydration {
...
+
   calculateAverageOunces(id) {
     let perDayUserHydration = this.hydrationData.filter((data) => id === data.userID);
-    return perDayUserHydration.reduce((sumSoFar, data) => {
+    return Math.round(perDayUserHydration.reduce((sumSoFar, data) => {
       return sumSoFar += data.numOunces;
-    }, 0) / perDayUserHydration.length;
+    }, 0) / perDayUserHydration.length);
   }
+
```
Comment (itsnameissteven) [inline src/Hydration.js:12]: Nice. I think math.round make sense here

File: test/hydration-test.js [hunk 4]
```diff
@@ -3,176 +3,104 @@ import Hydration from '../src/Hydration';
...
+  it('should find the water intake for a user on a specified date', () => {
+    expect(hydration.calculateDailyOunces(1, "2019/06/15")).to.equal(37);
+    expect(hydration.calculateDailyOunces(2, "2018/10/24")).to.equal(40);
+  });
+
+  //these last two functions aren't working properly
+  it('should find water intake by day for first week', () => {
+    expect(hydration.calculateFirstWeekOunces(userRepo, 1)).to.eql(['2019/06/15: 37', '2018/06/16: 39', '2016/08/22: 30']);
+    //this function needs the date as a parameter
+  });
```
Comment (itsnameissteven) [inline test/hydration-test.js:101]: I agree this is kind of a funky function and should be worked on for UI, if you trace back the function "getFirstWeek" it contains another function "makesortedUserArray"     and slicing the first 7 indexes in the array.  makeSorted sorts the original array by date .  this definately has some holes in the logic and calculateFirstWeek should probably be renamed to make it reflect what it is actually returning

**Review by itsnameissteven — APPROVED** [review]

**Pull request merged**